lib/router/index.d.ts

```diff
@@ -28,6 +28,12 @@ declare namespace createRouter {
         (req: Request, res: Response, next: NextFunction, value: any, name: string): any;
     }
 
+    export interface Options {
```

Comment:
Please name this `RouterOptions`, and export it also (as a type alias) in the index file.